vnext/src/Libraries/Components/DatePicker/DatePicker.tsx

```diff
@@ -16,7 +16,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const RCTDatePicker = requireNativeComponent('RCTDatePicker');
+const RCTDatePicker = requireNativeComponent<any>('RCTDatePicker');
```

Comment:
>any [](start = 45, length = 3)

This is a tad lazy, but the native component interface doesn't match the JS component, is nontrivial, and we're we're going to jettison this and reimplement it anyway.